src/openrct2/actions/FootpathSceneryPlaceAction.hpp

```diff
@@ -53,13 +53,12 @@ DEFINE_GAME_ACTION(FootpathSceneryPlaceAction, GAME_COMMAND_PLACE_FOOTPATH_SCENE
         auto res = MakeResult();
         res->ExpenditureType = RCT_EXPENDITURE_TYPE_LANDSCAPING;
         res->Position = _loc;
-        if (!map_is_location_valid({ _loc.x, _loc.y }))
+        if (!map_is_location_valid(_loc))
         {
             return MakeResult(GA_ERROR::INVALID_PARAMETERS, STR_CANT_POSITION_THIS_HERE, STR_OFF_EDGE_OF_MAP);
         }
 
-        if (!((gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR) || gCheatsSandboxMode)
-            && !map_is_location_owned(_loc.x, _loc.y, _loc.z / 8))
```

Comment:
You discarded the `/ 8` here.  In essence, this is correct (a function should either take big or small coordinates), but the calls to `map_is_location_owned()` will need to be carefully checked  to see if all of them now pass the `z` coordinate correctly.

Reply:
Yes I'm pretty confident I have all of these correct.